src/App.tsx

```diff
@@ -1,97 +1,94 @@
-import React from 'react';
+import React, {
+  useState, useMemo, useEffect, useCallback,
+} from 'react';
+import classNames from 'classnames';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { PersonList } from './components/PersonList/PersonList';
+import { InputPeople } from './components/InputPeople/InputPeople';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [selectedPeople, setSelectedPeople] = useState<Person | null>(null);
+  const [showList, setShowList] = useState(false);
+  const [pageTitle, setPageTitle] = useState<string>('No selected person');
 
-  return (
-    <div className="container">
-      <main className="section is-flex is-flex-direction-column">
-        <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
-        </h1>
-
-        <div className="dropdown is-active">
-          <div className="dropdown-trigger">
-            <input
-              type="text"
-              placeholder="Enter a part of the name"
-              className="input"
-              data-cy="search-input"
-            />
-          </div>
+  const applyQuery = useCallback(
+    debounce((inputQuery: string) => setAppliedQuery(inputQuery), 1000),
+    [setAppliedQuery],
+  );
 
-          <div
-            className="dropdown-menu"
-            role="menu"
-            data-cy="suggestions-list"
-          >
-            <div className="dropdown-content">
-              <div
-                className="dropdown-item"
-                data-cy="suggestion-item"
-              >
-                <p className="has-text-link">Pieter Haverbeke</p>
-              </div>
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
You can import ChangeEvent like useState from `'react'` and your type will be shorter

Reply:
<p
    className={classNames({
    'has-text-link': person.sex === 'm',
    'has-text-danger': person.sex === 'f',
     })}
 >
 this code I need to change?